x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeDataFrameAnalyticsIntegTestCase.java

```diff
@@ -67,6 +69,20 @@ private void cleanUpAnalytics() {
         }
     }
 
+    private void stopAnalyticsAndForceStopOnError() {
+        try {
+            assertThat(stopAnalytics("*").isStopped(), is(true));
+        } catch (Exception e) {
+            logger.error("Failed to stop data frame analytics jobs; trying force", e);
+            try {
+                assertThat(forceStopAnalytics("*").isStopped(), is(true));
+            } catch (Exception e2) {
+                logger.error("Force-stopping data frame analytics jobs failed", e);
```

Comment:
`e` is already logged in 2 other places so it would probably be more useful to log `e2` here.